app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -20,6 +20,9 @@
                         android:id="@+id/nav_qrcode"
                         android:icon="@drawable/ic_qrcode_24dp"
                         android:title="@string/qr_barcode_pdf" />
+                    <item
+                        android:id="@+id/nav_excel_to_pdf"
+                        android:title="@string/excel_to_pdf" />
```

Comment:
Add appropriate icon
Also change text in navigation drawer from `Excel to Pdf` to `Excel to PDF`.

Reply:
Okay